community/modules/scripts/wait-for-startup/scripts/wait-for-startup-status.sh

```diff
@@ -37,25 +37,26 @@ now=$(date +%s)
 deadline=$((now + TIMEOUT))
 error_file=$(mktemp)
 fetch_cmd="gcloud compute instances get-serial-port-output ${INSTANCE_NAME} --port 1 --zone ${ZONE} --project ${PROJECT_ID}"
+FINISH_LINE="startup-script exit status"
+FINISH_LINE_ERR="Script.*failed with error:"
```

Comment:
nit: I would give some context to future readers as to why we have two options. If you can think of some descriptive naming that could be good but `OLD_FINISH_LINE` and `NEW_FINISH_LINE` don't seem great. Maybe just a comment that states that the output has changed over time and this script supports both old and new versions.

Reply:
Done